pkg/reconciler/kubernetes/tektonresult/transform.go

```diff
@@ -271,6 +272,119 @@ func updateApiConfig(s v1alpha1.TektonResultSpec) mf.Transformer {
 	}
 }
 
+// update the api container envs
+func updateApiEnv(s v1alpha1.TektonResultSpec) mf.Transformer {
+	p := s.ResultsAPIProperties
+	return func(u *unstructured.Unstructured) error {
+
+		if u.GetKind() != "Deployment" || u.GetName() != deploymentAPI {
+			return nil
+		}
+
+		values := reflect.ValueOf(p)
+		types := reflect.TypeOf(p)
+		prop := make(map[string]string)
+
+		applyLokiStackConfig(prop, s.LokiStackProperties)
```

Comment:
what happens if an error happens on applyLokiStackConfig

Reply:
It doesn't apply any configuration and just adds LokiStack configuration details in the map